src/transactions/TransactionFrameBase.h

```diff
@@ -48,14 +49,18 @@ class TransactionFrameBase
     checkValid(Application& app, AbstractLedgerTxn& ltxOuter,
```

Comment:
It looks like the `ltx` checkValid variant is only used in tests, can we move this to the `TestTransactionFrame` class?

Reply:
good idea, done